lib/Form/Checkbox/Group.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from './';
+
+export default class Group extends Component {
+  static propTypes = {
+    choices: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
You can get away with just doing;

```
choices: PropTypes.arrayOf(PropTypes.shape()),
```

Unless you're providing details of the object structure.